tools/src/commands/GenerateBareApp.ts

```diff
@@ -0,0 +1,133 @@
+import path from 'path';
+import fs from 'fs-extra';
+import { Command } from '@expo/commander';
+import spawnAsync from '@expo/spawn-async';
+import { runExpoCliAsync } from '../ExpoCLI';
+
+type GenerateBareAppOptions = {
+  template: string;
+  clean?: boolean;
+  outDir?: string;
+};
+
+async function action(appName: string, packageNames: string[], options: GenerateBareAppOptions) {
+  function symlinkPackageManually(packageName: string) {
+    const packagePath = path.resolve(projectDir, 'node_modules', packageName);
+    if (fs.existsSync(packagePath)) {
+      fs.rmdirSync(path.resolve(projectDir, 'node_modules', packageName), { recursive: true });
+    }
+
+    fs.symlinkSync(path.resolve(repoRoot, 'packages', packageName), packagePath);
+  }
+
+  // TODO:
+  // if appName === ''
+  // if packageNames.length === 0
+  // configure autolinking to exclude dev-client / expo update packages
+
+  const { clean, outDir = 'bare-apps', template = 'expo-template-bare-minimum' } = options;
+
+  const repoRoot = path.resolve(__dirname, '../../../');
+  const projectsDir = path.resolve(process.cwd(), outDir);
+  const projectDir = path.resolve(process.cwd(), projectsDir, appName);
+
+  if (!fs.existsSync(projectsDir)) {
+    fs.mkdirSync(projectsDir);
+  }
+
+  if (clean) {
+    if (fs.existsSync(projectDir)) {
+      fs.rmdirSync(projectDir, { recursive: true });
+    }
+  }
+
+  // these don't seem to symlink properly w/ expo-yarn-workspaces
+  const symlinkedPackages = ['expo', 'expo-modules-autolinking'];
+
+  // these seem to be included regardless of what is in node_modules (default packages for prebuild)
+  const defaultPackages = [
+    'expo-asset',
+    'expo-application',
+    'expo-constants',
+    'expo-file-system',
+    'expo-font',
+    'expo-keep-awake',
+    'expo-error-recovery',
+    'expo-modules-core',
+    'expo-splash-screen',
+    'expo-updates',
+    'expo-dev-client',
+    'expo-dev-launcher',
+    'expo-dev-menu',
+    'expo-dev-menu-interface',
+  ];
+
+  await runExpoCliAsync('init', [appName, '--no-install', '--template', template], {
+    cwd: projectsDir,
+  });
+
+  const pkg = require(path.resolve(projectDir, 'package.json'));
+
+  pkg['expo'] = pkg['expo'] ?? {};
+
+  // TODO - autolinking -> { include?: [] }
+  // pkg['expo']['autolinking'] = { exclude: [] };
+
```

Comment:
apologies! I was thinking of having an "include" API which would allow a manual override to auto linking -- there are certain modules that are included by default in every project (e.g dev-launcher) that I'd like to explicitly omit because I know I don't need them, but prebuild thinks I do (because monorepo)

I thought that the "exclude" field would achieve this, however the prebuild command uses different logic for determining which modules are present (they do not read from this auto linking field in `package.json`)

Reply:
I don't think there is a workaround for this at the moment so I might just remove the comment altogether